.github/workflow-scripts/gh-pages/download-branch-deploy-artefact.js

```diff
@@ -0,0 +1,25 @@
+module.exports = async ({
+  sha,
+  github,
+  context: { repo: { repo, owner } },
+}) => {
+  // get all workflow runs for the repo
```

Comment:
This comment seems superfluous to me

Reply:
Yes, good point - i think it's left over from when I was thinking about the process. I'll remove it 👍

Reply:
Updated 👍